src/styles/global/colors/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import colors from './_index.scss'
+
+const Color = () => {
```

Comment:
While it still works because it's the default export, you're calling it `Colors` in the docs and `Color` here. I'd probably go with the plural just to be consistent.